baseplate/lib/edge_context.py

```diff
@@ -92,7 +104,7 @@ def subject(self) -> Optional[str]:
 
     @cached_property
     def user_roles(self) -> Set[str]:  # type: ignore # pylint: disable=W0236
-        return set(self.payload.get("roles", []))
+        return set(self.payload.get("roles") or [])
```

Comment:
why is this changing?

Reply:
what if pass the role as None then it will get the errors if we set(None)

Reply:
but is that changing, or was that always possible (and not causing problems)?